FlagEditor/FlagEditor.js

```diff
@@ -0,0 +1,125 @@
+const fsx = require('fs-extra');
+const path = require('path');
+
+const getGitUsername = require('./gitTools/git-user-name');
+
+const { ACTIONS, ENVIRONMENTS } = require('./constants');
+
+const FLAGS_FILENAME = 'NEARFlags.json';
+const DEBUG_LOGGING = false;
+
+class FlagEditor {
```

Comment:
seems like this can be separate open source npm package vs built into wallet

Reply:
**Absolutely**.  For initial implementation I'd like to get it reviewed and tested here, then move it into its own NPM package so that it could be used for any JS ecosystem packages at NEAR (or elsewhere, with some basic configuration!)